### DISCUSSION TITLE
Erro na API

### DISCUSSION THREAD

**Discussion by pedro-soares-engix** [filipeas/boleto-banco-brasil#5] [Q&A] [answered]
Estou tendo um erro dentro da API, porém não sei bem como resolver:

C:\Users\P.Soares\Desktop\gerador de boletos\node_modules\boleto-banco-brasil\dist\adapters\bb-client.js:259
    const valor = data.ticketValue.toLocaleString('pt-br', {
                                                            ^

TypeError: Cannot read properties of undefined (reading 'toLocaleString')
    at BBClient.CreatePurchaseTicket (C:\Users\P.Soares\Desktop\gerador de boletos\node_modules\boleto-banco-brasil\dist\adapters\bb-client.js:259:38)
    at C:\Users\P.Soares\Desktop\gerador de boletos\app.js:119:34
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v20.5.0

**Comment by filipeas** [comment, marked as answer]
Olá.

Se possível, coloca um exemplo de como você está utilizando a função CreatePurchaseTicket. O erro pode estar em um dos parametros informados.

No mais, dá uma olhada nos exemplos de como utilizar para verificar se tem alguma coisa faltando ou coisa do tipo.
